itl/plugins-contrib.d/ipmi.conf

```diff
@@ -111,7 +111,7 @@ object CheckCommand "ipmi-sensor" {
 
 object CheckCommand "ipmi-alive" {
         import "plugin-check-command"
-        command = [ PluginContribDir + "/check_ping" ]
+        command = [ PluginDir + "/check_ping" ]
```

Comment:
As it is check_ping PluginDir would be correct, but perhaps importing ping similar to hostalive would be better?